### PR TITLE
Add table Pluggable vs Dojo

### PR DESCRIPTION
**Pull request by Andries-Smit**

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md [hunk 1]
```diff
@@ -19,6 +19,25 @@ Pluggable widgets can also include a preview component for when they are rendere
 
 Pluggable widgets are the successor to [Custom widgets](/howto/extensibility/widget-development). Pluggable widgets are based on a modern React framework, have access to better APIs in the client, and can use more advanced properties in Studio and Studio Pro. Therefore, pluggable widgets allow you to more easily implement existing front-end libraries, tools, and your own expertise. Pluggable widgets should be used instead of custom widgets whenever possible.
 
+| Widget | Pluggable | Custom | Description |
+| --- | --- | --- | --- |
+| Technology | [React](#3-client-component) | Dojo | Rendering |
+| API | Declarative | Asynchronous | |
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: - API -> Data access
- Declarative -> Declarative, props-based
- Asynchronous -> Imperative, callback-based

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md [hunk 2]
```diff
@@ -19,6 +19,25 @@ Pluggable widgets can also include a preview component for when they are rendere
 
 Pluggable widgets are the successor to [Custom widgets](/howto/extensibility/widget-development). Pluggable widgets are based on a modern React framework, have access to better APIs in the client, and can use more advanced properties in Studio and Studio Pro. Therefore, pluggable widgets allow you to more easily implement existing front-end libraries, tools, and your own expertise. Pluggable widgets should be used instead of custom widgets whenever possible.
 
+| Widget | Pluggable | Custom | Description |
+| --- | --- | --- | --- |
+| Technology | [React](#3-client-component) | Dojo | Rendering |
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: Framework? UI technology?

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md [hunk 3]
```diff
@@ -19,6 +19,25 @@ Pluggable widgets can also include a preview component for when they are rendere
 
 Pluggable widgets are the successor to [Custom widgets](/howto/extensibility/widget-development). Pluggable widgets are based on a modern React framework, have access to better APIs in the client, and can use more advanced properties in Studio and Studio Pro. Therefore, pluggable widgets allow you to more easily implement existing front-end libraries, tools, and your own expertise. Pluggable widgets should be used instead of custom widgets whenever possible.
 
+| Widget | Pluggable | Custom | Description |
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: I don't see much value of description column. Maybe better use its text in a first column to make that more explicit.

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md (hunk 1 above)
Comment (mijay) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: It is actually really vague row and it overlaps with the next 3 a lot...

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md [hunk 4]
```diff
@@ -19,6 +19,25 @@ Pluggable widgets can also include a preview component for when they are rendere
...
+| --- | --- | --- | --- |
+| Technology | [React](#3-client-component) | Dojo | Rendering |
+| API | Declarative | Asynchronous | |
+| Data | Receive | Retrieve | Pluggable data is received as properties |
+| [Data attribute](/property-types-pluggable-widgets#attribute) | Yes | Yes | Access the content of the data view |
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: It is more "attribute access". And link is better in a column, not in the caption. The latter appllies to other rows as well

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md [hunk 5]
```diff
@@ -19,6 +19,25 @@ Pluggable widgets can also include a preview component for when they are rendere
...
+| Data source | Planned | Yes | Retrieve data database, association, microflow or nanoflow |
+| Containment | Planned | No | Widget can contain other widgets |
+| [Template Text](property-types-pluggable-widgets#texttemplate) | Yes | No | Custom widget can use translatable strings |
+| [On Change Action](/property-types-pluggable-widgets#attribute) | Yes | No | Trigger a action on attribute change |
+| [Visibility](property-types-pluggable-widgets#visibility) | Yes | No | Will have system property for visibility |
+| [Editability](property-types-pluggable-widgets#editability) | Yes | No | Properties editable, custom widget can not use expressions |
+| [Expression](property-types-pluggable-widgets#expression) | Yes | No | |
+| [Client API](https://apidocs.mendix.com/8/client/) | No | Yes | Use mendix, mxui or mx client API |
+| [Native](#5-2-widget-description) | Yes | No | Build Native Client widgets |
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: I think rows of this table are quite unaligned. Some things are small features, some are major, and some are conceptual differences.

Comment (Andries-Smit) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: Will splint up in 3 tables: concepts, data and features

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md (hunk 4 above)
Comment (Andries-Smit) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: removed data row, reworded data source row

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md (hunk 1 above)
Comment (Andries-Smit) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: changed

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md (hunk 2 above)
Comment (Andries-Smit) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: =>  UI technology

File: content/apidocs-mxsdk/apidocs/pluggable-widgets.md (hunk 3 above)
Comment (Andries-Smit) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets.md]: Done

**Pull request merged**